### DISCUSSION TITLE
Provide an option to disable "Help people interested in this repository understand your project by adding a README." callout

### DISCUSSION THREAD

**Discussion by levicki** [community/community#51124] [Repositories]
### Select Topic Area

Product Feedback

### Body

To be clear I am referring to this callout:

![image](https://user-images.githubusercontent.com/16415478/227956711-0ca9168b-6d28-4636-a73a-22b7eddcbff1.png)

I understand the intent is to nudge people to always include `README.md` file in the repository, but I do not believe in the "one size fits all" approach when it comes to software development workflows.

I might decide to have a `README.pdf` in the repository under `DOCS` folder, or I might use Wiki pages, not to mention that I might be working alone on it, or that the repository might contain different content that doesn't require explanation, or it may even consist entirely of self-explanatory content such as various aptly named documents or resource files and zero source code.

TL;DR &mdash; Sometimes having `README.md` makes no sense, and having this callout always visible if there's no `README.md` file in the repository is extremely visually distracting when I am supposed to be looking at the source tree.

It sits there begging for attention with its distinctive background color and screaming green button creating a feeling of "you left something unfinished!" which is a serious OCD trigger at least for me.

Finally, I find it quite rude when people believe their practice is the best practice and enforce it on others without the ability to opt-out.

Microsoft developers and product managers seems to be most guilty of doing just that, and it affects me regardless of whether I am using developer tools such as GitHub and Visual Studio or end user stuff like Windows and Office apps.

Please add some option to Settings to disable that callout.

Thanks in advance.
